package.json

```diff
@@ -17,7 +17,7 @@
     "auth": "npm_config_registry=https://registry.npmjs.org npm whoami",
     "clean": "lerna exec 'rm -rf dist'",
     "generate-readmes": "yarn generate-readme --all-workspace-packages",
-    "generate-icons": "node ./scripts/generate-icon-exports.js",
+    "generate-icons": "lerna run generate-icons && prettier --write '**/icons/components/*.js'",
```

Comment:
🤔 do we need lerna here or could we use yarn workspaces to run it?

Reply:
yarn workspaces fails if a package does not have the script defined. Lerna is more smart about it and only runs the command for the packages that contains the given script.

Reply:
Oki doks. I was just hoping we could get rid of lerna by e.g. having a `exit x` or something in all scripts.

Reply:
We could but it's still not really better because yarn workspaces will run in all packages anyway, while lerna only runs to the few packages that have the command.

What we can do is to have our own "workspace" script, using the `manypkg` API to inspect the packages.